api/src/scripts/populate_db.py

```diff
@@ -224,6 +192,8 @@ def add_entity(entity, priority):
             )
             add_entity(mdb_external_id, 4)
 
+        [add_entity(entity_type, 4) for entity_type in entity_types.values()]
```

Comment:
This creates an list, right? Why bother if we don't do anything with it.
Can't we use a simple loop?

Reply:
this is just to save some space the list isn't saved to a variable

Reply:
But it's taking memory and time for no good reason. 
I admit though that it might be negligible.